website/src/css/custom.css

```diff
@@ -2560,11 +2489,68 @@ iframe#q-messenger-frame:not(.qlfd-maximized) {
 
 @media (max-width: 996px) {
   .baton-1::before,
-  .baton-2::before {
+  .baton-2::before,
+  .baton-hero::before {
     background-size: contain;
   }
 }
 
+.baton-hero {
+  text-align: center;
+  padding: 6rem 1rem 20rem;
+}
+
+@media (max-width: 996px) {
+  .baton-hero {
+    padding: 4rem 1rem 6rem;
+  }
+}
+
+.baton-hero .container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.hero-search-input input {
+  max-width: 400px;
+  width: 100%;
+  padding: .75rem;
+  height: 48px;
+  margin-top: 1rem;
+  font-family: var(--font-sans);
+  background: var(--bg-neutral-primary-default);
+  border: 1px solid var(--bg-neutral-secondary-default);
+  cursor: pointer;
+  border-radius: 12px;
+}
```

Comment:
In darkmode, the text in the hero search box changes to white and becomes invisible

![image](https://github.com/user-attachments/assets/4a351465-00f0-4f0b-b8e2-b5151c352ee3)

Reply:
Can we also add a hover state for the quick links? My initial reaction is the underline should disappear on hover.

Reply:
good catch, both should be fixed now.